docs/output.md

```diff
@@ -112,20 +112,21 @@ For further reading and documentation see the [FastQC help](http://www.bioinform
 
-A section of the bar will also show an approximate estimation of the fraction of the total number of reads that are duplicates of another. This can derive from over-amplifcation of the library, or lots of single adapters. This can be later checked with the Deduplication check. A good library and sequencing run should have very low amounts of duplicates reads.
+A section of the bar will also show an approximate estimation of the fraction of the total number of reads that are duplicates of another. This can derive from over-amplification of the library, or lots of single adapters. This can be later checked with the Deduplication check. A good library and sequencing run should have very low amounts of duplicates reads.
 
 <p align="center">
-  <img src="images/output/fastqc/sequence_counts.png" width="75%" height = "75%">
+  <img src="images/output/fastqc/fastqc_sequence_counts.png" width="75%" height = "75%">
 </p>
 
 #### Sequence Quality Histograms
 
 This line plot represents the Phred scores across each base pair of all the reads. The x-axis is the base position across each read, and the y-axis is the average base-calling score (Phred-scaled) of the nucleotides across all reads. Again, this is per FASTQ file (i.e. forward/reverse and/or lanes separately). The background colours represent approximate ranges of quality, with green section being acceptable quality, orange is dubious and red is bad.
 
-You will often see that the first 5 or so bases have slightly lower quality than the rest of the read as this the calibration steps of the machine. The bulk of the read should then stay ~35. Do not worry if you see the last 10-20 bases of reads do often have lower quality base calls that the middle of the read, as the sequencing reagents start to deplete during these cycles (e.g. making nucleotide flourescence weaker). Furthermore, the reverse reads of sequencing data will often be even lower at ends than forward reads for the same reason.
+You will often see that the first 5 or so bases have slightly lower quality than the rest of the read as this the calibration steps of the machine. The bulk of the read should then stay ~35. Do not worry if you see the last 10-20 bases of reads do often have lower quality base calls that the middle of the read, as the sequencing reagents start to deplete during these cycles (e.g. making nucleotide fluorescence weaker). Furthermore, the reverse reads of sequencing data will often be even lower at ends than forward reads for the same reason.
 
 <p align="center">
-  <img src="images/output/fastqc/sequencing_quality_histogram.png" width="75%" height = "75%">
+  <img src="images/output/fastqc/fastqc_sequence_quality_histogram.png
```

Comment:
This one doesn't render properly - see https://github.com/jfy133/eager/blob/tsv-input/docs/output.md